### PR TITLE
Crez updates to item_display and building_facet

### LINKED ISSUE #59: Modify crez fields
Crez indexing modifies `item_display` and `building_facet`

See https://github.com/sul-dlss/crez-sw-ingest/blob/master/lib/add_crez_to_solr_doc.rb

Prerequisites: #22

A spin off ticket from #27

### PR DESCRIPTION
**Pull request by camillevilla**
Fixes #59

### CODE REVIEW

File: lib/translation_maps/library_code_translations.rb [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+{
```
Comment (cbeer) [inline lib/translation_maps/library_code_translations.rb]: This seems suspiciously like our `library_map.properties` file?

Comment (mejackreed) [reply lib/translation_maps/library_code_translations.rb]: Yep. Looks like there are two minor differences. Should we consolidate now?

Comment (mejackreed) [reply lib/translation_maps/library_code_translations.rb]: Though I wonder if the "lack of" the 13 additional keys at the end could cause problems.. I think that is why we just ported over what we had for now.

Comment (cbeer) [reply lib/translation_maps/library_code_translations.rb]: Ah, I didn't scroll down to the bottom. I'll push the green button and let @jkeck chime in next week.

**Pull request merged**